src/form/Form.tsx

```diff
@@ -346,23 +343,17 @@ function Form({
     );
     const submitButton = renderedButtons.find(
       (b: any) =>
-        b.properties.link === LINK_NEXT && !b.properties.link_no_submit
+        b.properties.link === LINK_SEND_SMS && !b.properties.link_no_submit
+    );
+    const smsButton = renderedButtons.find(
+      (b: any) => b.properties.link === LINK_SEND_SMS
     );
-    if (hasLoginField && submitButton) {
+
+    if ((hasPhoneField && submitButton) || smsButton) {
```

Comment:
I tried to keep this conditionals equivalent to the old code, but I think I can just check for `smsButton`?

Reply:
yes you can just check for smsButton